comparators/invalid-tag-modification.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = invalidTagModification;
+
+
+function getPrimaryTag(properties) {
+  var primaryTags = [
+    'aerialway', 'aeroway', 'amenity', 'barrier', 'boundary', 'building', 'craft', 'emergency',
+    'geological', 'highway', 'historic', 'landuse', 'leisure', 'man_made', 'military', 'natural',
+    'office', 'places', 'power', 'public_transport', 'railway', 'route', 'shop', 'sport',
+    'tourism', 'waterway'
+  ];
+  for (var key in properties) {
+    if (primaryTags.indexOf(key) !== -1) return key;
```

Comment:
@amishas157 I wanted to keep this compare function handle just the case when there is one primary tag. As you said, we have another compare function to flag when there are multiple primary tags in a feature that are unpopular.